tests/containers/podman_network.pm

```diff
@@ -52,10 +52,36 @@ sub run() {
         assert_script_run('podman network inspect newnet1 --format "{{range .Subnets}}Subnet: {{.Subnet}} Gateway: {{.Gateway}}{{end}}"');
     }
 
+    #connect, disconnect & reload
+    unless ($old_podman) {
```

Comment:
`$old_podman` is not really descriptive. Petition to rename it to `$supports_network` or something similar, which is more descriptive.

Why? `old` can mean anything, `supports_network` suggests already what the purpose of the variable is.